ruby/regex_literals.md

```diff
@@ -0,0 +1,19 @@
+# Regex Literals
+
+Regular expressions in Ruby are typically delineated by the `/` character.
+However, this forces you to escape `/` in your expression.
+
+I was recently using a regex to match certain URLs
+
+```ruby
+/http:\/\/sub\.domain\.com\/path/
+```
+
+A colleague pointed out that I could use the `%r{...}` notation and avoid
```

Comment:
You can use any punctuation that works with `%w`:

* `%r()`
* `%r||`
* %r<>`

If you want to get wacky, any punctuation created by shift-NUMBER works:

* `%r!!`
* %r@@`

etc.